src/main.cpp

```diff
@@ -1,6 +1,10 @@
-#include "common.hpp"
+#include <csignal>
+#include <iostream>
+#include <thread>
+#include <vector>
 
-using json = nlohmann::json;
```

Comment:
`nlohmann::json` and `json` were unused in this file.

Reply:
This was a remnant of a previous iteration of the project that I forgot to remove. Good one.